internal/store/device.go

```diff
@@ -17,6 +17,30 @@ import (
 	"gorm.io/gorm"
 )
 
+// DeviceStatusType represents the type of device status to query
```

Comment:
Please add integration tests for all of the new methods in test/integration/store/

Reply:
done